### PR TITLE
Added authentication message template with OTP

### PR DESCRIPTION
**Pull request by LaErika**
It adds functionality for sending authentication messages with OTP template with option Copy Code button

### CODE REVIEW

File: lib/api/messages.ex [hunk 1]
```diff
@@ -49,6 +50,24 @@ defmodule Whatsapp.Api.Messages do
     end
   end
 
+  def send_hsm_otp({:error, _} = error, _), do: error
+
+  def send_hsm_otp({url, auth_header}, %MessageOutboundHsmOtp{} = message) do
+    IO.inspect("WAX SEND HSM OTP")
```
Comment (MisaelRaLo) [inline lib/api/messages.ex:56]:
```suggestion

```

File: lib/api/messages.ex [hunk 2]
```diff
@@ -49,6 +50,24 @@ defmodule Whatsapp.Api.Messages do
...
+
+    with {:ok, message_validated} <- MessageOutboundHsmOtp.validate(message) do
+      headers = [auth_header]
+
+      message = MessageOutboundHsmOtp.to_json(message_validated)
+      IO.inspect(message)
```
Comment (MisaelRaLo) [inline lib/api/messages.ex:62]:
```suggestion
  
```

**Review by MisaelRaLo — CHANGES_REQUESTED** [review]

File: lib/api/messages.ex (hunk 1 above)
Comment (LaErika) [reply lib/api/messages.ex]: Se resolvio en [cc91168](https://github.com/resuelve/wax/pull/72/commits/cc911683303d49195300482141e8d05d8547d8ae)

File: lib/api/messages.ex (hunk 2 above)
Comment (LaErika) [reply lib/api/messages.ex]: Se resolvio en [cc91168](https://github.com/resuelve/wax/pull/72/commits/cc911683303d49195300482141e8d05d8547d8ae)

**Review by MisaelRaLo — APPROVED** [review]

**Pull request merged**